v2/inc/c_logging/log_context.h

```diff
@@ -4,6 +4,241 @@
 #ifndef LOG_CONTEXT_H
 #define LOG_CONTEXT_H
 
+#ifdef __cplusplus
+#include <cinttypes>
+#include <cstdio>
+#include <cstdlib>
+#else
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+#endif
+
+#include "macro_utils/macro_utils.h"
+
+#include "c_logging/log_context_property_value_pair.h"
+#include "c_logging/log_context_property_basic_types.h"
+#include "c_logging/log_context_property_type_ascii_char_ptr.h"
+#include "c_logging/log_context_property_type_struct.h"
+#include "c_logging/log_internal_error.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define LOG_MAX_STACK_DATA_SIZE                 1024
```

Comment:
>1024 [](http://example.com/codeflow?start=48&length=4)

I am going to say right now that our data logs sometimes exceed 4096 (previous limit).

Specially when logging those nasty cryptology data types.

Reply:
Made it 4K